include/envoy/common/platform.h

```diff
@@ -142,6 +143,9 @@ struct msghdr {
 #define SOCKET_ERROR_INVAL WSAEINVAL
 #define SOCKET_ERROR_ADDR_IN_USE WSAEADDRINUSE
 
+namespace platform {
```

Comment:
Even though the Google style guide mentions using lowercase namespace names, seems we have capitalized the first letter all throughout Envoy

Reply:
Thanks for pointing it out I fixed it